### PR TITLE
flatpak: Add clipboard support.

### LINKED ISSUE #3009: Issuing command `nyxt-version` on Flatpak fails
It lacks a clipboard program such as `xclip` or `xsel`. Needs to be added to the Flatpak package declaration.

Perhaps we should wrap `(trivial-clipboard:text +version+)` with `ignore-errors` nonetheless.

### LINKED ISSUE #3021: Clipboard support for Flatpak
We have made a crude mistake when we started shipping Nyxt via Flatpak since we forgot that we rely on CL library `trivial-clipboard`, which requires either `xclip` or `xsel`.

### PR DESCRIPTION
**Pull request by aadcg**
# Description

Tested - works fine.

Fixes #3021

Fixes #3009 

# Discussion

@jmercouris since this is a critical bug, I'd suggest releasing Nyxt 3.2.1 next Monday with this fix.

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.

- [x] Git hygiene:
  - I have pulled from master before submitting this PR
  - There are no merge conflicts.
- [x] I've added the new dependencies as:
  - ASDF dependencies,
  - Git submodules,
    ```sh
	cd /path/to/nyxt/checkout
    git submodule add https://gitlab.common-lisp.net/nyxt/py-configparser _build/py-configparser
    ```
  - and Guix dependencies.
- [x] My code follows the style guidelines for Common Lisp code. See:
  - [Norvig & Pitman's Tutorial on Good Lisp Programming Style (PDF)](https://www.cs.umd.edu/~nau/cmsc421/norvig-lisp-style.pdf)
  - [Google Common Lisp Style Guide](https://google.github.io/styleguide/lispguide.xml)
- [x] I have performed a self-review of my own code.
- [ ] My code has been reviewed by at least one peer.  (The peer review to approve a PR counts.  The reviewer must download and test the code.)
- [x] Documentation:
  - All my code has docstrings and `:documentation`s written in the aforementioned style.  (It's OK to skip the docstring for really trivial parts.)
  - I have updated the existing documentation to match my changes.
  - I have commented my code in hard-to-understand areas.
  - I have updated the `changelog.lisp` with my changes if it's anything user-facing (new features, important bug fix, compatibility breakage).
  - I have added a `migration.lisp` entry for all compatibility-breaking changes.
  - (If this changes something about the features showcased on Nyxt website) I have these changes described in the new/existing article at Nyxt website or will notify one of maintainters to do so.
- [x] Compilation and tests:
  - My changes generate no new warnings.
  - I have added tests that prove my fix is effective or that my feature works.  (If possible.)
  - I ran the tests locally (`(asdf:test-system :nyxt)` and `(asdf:test-system :nyxt/gi-gtk)`) and they pass.

### CODE REVIEW

**Comment by Ambrevar** [comment]
Looks good to me.

File: engineer.atlas.Nyxt.yaml [hunk 1]
```diff
@@ -58,6 +65,6 @@ modules:
         LISP_FLAGS: "--dynamic-space-size 4096 --no-userinit --non-interactive"
     build-commands:
       - make all
-      - mkdir $FLATPAK_DEST/bin
+      - mkdir -p $FLATPAK_DEST/bin
```
Comment (jmercouris) [inline engineer.atlas.Nyxt.yaml:68]: good ol' -p flag ;-)

**Comment by jmercouris** [comment]
Thank you Andre. I agree that we should release the fix.

**Comment by ntzm** [comment]
How easy would it be to support wl-copy,wl-paste for flatpak in Wayland?

**Comment by ntzm** [comment]
@aadcg yes I can :)

**Pull request merged**